network/local.go

```diff
@@ -11,25 +11,25 @@ import (
 )
 
 // NewLocalRouter returns a fresh router which uses local connections. It takes
-// the default context.
+// the default manager.
 func NewLocalRouter(sid *ServerIdentity) (*Router, error) {
-	return NewLocalRouterWithContext(defaultLocalContext, sid)
+	return NewLocalRouterWithManager(defaultLocalManager, sid)
 }
 
-// NewLocalRouterWithContext is the same as NewLocalRouter but takes a specific
-// LocalContext. This is useful to run parallel different local overlays.
-func NewLocalRouterWithContext(ctx *LocalContext, sid *ServerIdentity) (*Router, error) {
-	h, err := NewLocalHostWithContext(ctx, sid.Address)
+// NewLocalRouterWithmanager is the same as NewLocalRouter but takes a specific
```

Comment:
(To save you some time later) This might not pass the linter: Comment should start with NewLocalRouterWithManager